mod/thewire/languages/ru.php

```diff
@@ -13,7 +13,7 @@
 	'thewire' => "Микроблог",
 
 	'item:object:thewire' => "Микроблог",
-	'collection:object:thewire' => 'Wire posts',
+	'collection:object:thewire' => 'Записи микроблога',
```

Comment:
Don't make non English translations. These will get lost when we merge the translations from Transifex.

Just add the English translations and after the merge translate it on Transifex, when we release all translations get merged.

Reply:
Forgot about it.